### DISCUSSION TITLE
Location-Based Marginal Carbon Intensity Measurement (I): needs justification, and improved clarity and phrasing

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#157] [General]
The section includes undefined terminology "regional balancing authority", "marginal power plant", "schedule some compute".

I disagree with the statement "From a developer perspective, only the location-based info is important for having an impact on reducing carbon emissions." - a persistent reduction of electricity consumption attributable to a specific functional unit of software will contribute to a reduction of electricity use over a longer time, independent from the marginal effect.

**Comment by atg-abhishek** [comment]
Better suited for a discussion @dschien

**Comment by Henry-WattTime** [comment]
I agree, this would be a long term marginal effect. It would ultimately affect utility planning and power plant construction/decommissioning to create a persistent effect. We'd call that build margin or long run marginal effect.
